capacity/encode_change_request.js

```diff
@@ -51,5 +52,12 @@ module.exports = ({channel, decrease, id, increase}) => {
     });
   }
 
+  if(!!new_channel_type) {
+    records.push({
+      type: publicPrivateRecordType,
+      value: new_channel_type === 'public' ? encodeBigSize({number: 1337}).encoded : encodeBigSize({number: 1339}).encoded,
```

Comment:
If private/public is switched to channel flags to match `channel_flags` (type of channel data type) then it would look more like this: https://github.com/alexbosworth/lightning/blob/master/lnd_responses/channel_accept_as_open_request.js#L117 - private/public is encoded into a bit flip